scripts/SREP/update-upgrade-policy-state/script.sh

```diff
@@ -26,58 +26,60 @@ get_cluster_uuid() {
 # Params:
 #   - Cluster UUID
 get_cluster_id() {
-    ocm list clusters --parameter search="external_id is '${1}'" --columns id --padding 20 --no-headers
+    oc get configmap cluster-config-v1 -n kube-system -o jsonpath='{.data.install-config}' | grep -oE 'api.openshift.com/id: [^ ]+' | awk '{print $2}'
```

Comment:
do you need to update the metadata to include more permission if the new resource request added?

Reply:
@Dee-6777 - Bo's point is that you've updated the `get_cluster_id()` to get clusterID from `configmap` resource in the `kube-system` namespace instead of using the previous `ocm list clusters`, it's not something in our current RBAC definition in the metadata that allowing this script to be run as a job under a service account.

btw, can you clarify why you changed the previous way that uses ocm to get cluster id but configmap?